test/views/html/coc/HistorySummaryTest.scala

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
is this just not needed?

Reply:
It was a view test that I created early on. I have removed it, though there will be a ticket to create view tests and to remove the content controller tests.

Reply:
was this previuosly tested and now not tested at all then?

Reply:
It will be tested in the controller content test